tool/firestore-emulator/Dockerfile

```diff
@@ -1,19 +1,13 @@
-FROM openjdk:8-jre-alpine3.9
+ARG GOOGLE_CLOUD_SDK_VERSION=392.0.0-alpine
 
-ARG GOOGLE_CLOUD_SDK_VERSION=392.0.0
+FROM google/cloud-sdk:$GOOGLE_CLOUD_SDK_VERSION
 
-RUN apk add --no-cache \
-        python3 \
-        curl && \
-    curl -O https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-$GOOGLE_CLOUD_SDK_VERSION-linux-x86.tar.gz && \
-    tar -C /usr/local -xvzf google-cloud-sdk-$GOOGLE_CLOUD_SDK_VERSION-linux-x86.tar.gz && \
-    rm google-cloud-sdk-$GOOGLE_CLOUD_SDK_VERSION-linux-x86.tar.gz && \
-    /usr/local/google-cloud-sdk/install.sh --quiet
+RUN apk add --update --no-cache openjdk11-jre-headless
 
-ENV PATH $PATH:/usr/local/google-cloud-sdk/bin
+ENV FIRESTORE_PROJECT_ID "project"
```

Comment:
How about making this more specific? For example, `pipecd-test`

Reply:
Thank you. it's better.